### ISSUE TITLE
Disable debug output on travis?

### ISSUE THREAD

**Issue by matthiasbeyer** [matthiasbeyer/imag#658]
When running our test scripts on travis, they produce a _huge_ amount of debug output.

Maybe we should disable these as well, so we get shorter output on travis? I'm not sure.

@TheNeikos , @neithernut what do you think?

**Comment by neithernut** [comment]
IMO, debug options are meant for when the user has a problem. Typically, she may consult the typical support channel and ask for help. If the problem is obvious enough and reproducible, the mainainter/programmer may go from there and fix the bug. If the problem is not that obvious, the user may be asked to try reproduce the bug again with the debug flag set to true, hopefully producing output containing some hint on what the problem might be.

The debug output is only of interest if something goes wrong. One might argue that having the debug output once the CI fails cannot hurt. The amount of messages ending up in the log might, however, burry some clear error message pointing at the exact problem. Also, as the CI runs are supposed to be 100% reproducible, we can always temporarily enable the debug flag should we see a bug which fails to reproduce on some other machine.

TL;DR: disable the debug flag.

**Comment by TheNeikos** [comment]
I was actually for keeping it enabled, but @neithernut is correct in arguing that they should be reproducible, so if something breaks there we can just retry it locally and (hopefully) see the same thing.

So I am also for disabling them.

**Comment by matthiasbeyer** [comment]
Ok. I removed the 'decision-pending' label ... this is now a tracking issue.

**Comment by matthiasbeyer** [comment]
@asuivelentine a job for you maybe? :smile:
